l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot

```diff
@@ -0,0 +1,130 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* l10n_nl_dutch_company_type
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_nl_dutch_company_type
+#: model:res.partner.company.type,name:l10n_nl_dutch_company_type.company_type_association
+msgid "Association"
+msgstr ""
```

Comment:
no translation?

Reply:
No translation needed here, it is the pot file, translation are in nl.po